pyvo/registry/rtcons.py

```diff
@@ -719,9 +719,16 @@ def tomoc(s):
 
         elif len(geom_spec) == 2:
             if isinstance(geom_spec[0], SkyCoord):
+                # If radius given is astropy quantity, then convert to degrees
```

Comment:
Please add this case to the docstring, and possibly an example to the narrative docs

Reply:
Tried my best to write useful examples!

Docstring example implemented in: https://github.com/astropy/pyvo/pull/594/commits/be70c436e21324b5c3cac208774b4d5a93f25956

Narrative example implemented in: https://github.com/astropy/pyvo/pull/594/commits/d91c80a4ee89ee13aacce9425b0e61dd34cea7cd

How do those look?

Reply:
Re narrative docs: There is value to list the expected outputs, so I'll add those to your example, as well as will remove the IGNORE_OUTPUT, as we use that only for cases where results are expected to chang often or are unpredictable.

Reply:
(ahh, never mind, I see that we already have that in the previous example)